studio/src/components/Timeline/hooks/useAsWaterfall.ts

```diff
@@ -39,6 +39,17 @@ export function useAsWaterfall(
         if ("span" in b && b?.span?.name === "request") {
           return 1;
         }
+
+        // If the time stamp is the same, sort on span_id/parent_span_id
+        // TODO: improve further sorting.
+        if ("span" in a && "span" in b) {
```

Comment:
ah nice — i saw on monday that with goose quotes, sometimes the first child span rendered above the root request span, forgot to file a ticket. guessing this fixes that!